websites/W/watch.lonelil.com/metadata.json

```diff
@@ -20,12 +26,13 @@
 		"movie",
 		"tv",
 		"show",
-		"anime"
+		"anime",
+		"pokemon"
 	],
 	"settings": [
 		{
 			"id": "lang",
-			"multiLanguage": "kahoot"
+			"multiLanguage": true
```

Comment:
> Got it. Was confused about what that meant/did. What if something uses strings from multiple websites?

It can also be an array of strings
https://docs.premid.app/en/dev/presence/metadata#multilanguage